department-of-veterans-affairs/va.gov-cms
# VACMS-5347: Fix hub landing proofing pages.

## Description

See #5347. 

## Testing done

Manual / visual

## Screenshots

![VA_Health_Care___Veterans_Affairs_and_VA_Health_Care___Veterans_Affairs](https://user-images.githubusercontent.com/643678/118523829-87744780-b70b-11eb-9559-81e57f5b6163.png)

## QA steps

1. Go to /health-care 
2. You should see a linked label to VHA rather than the layout builder rendered entity

### Definition of Done


### Select Team for PR review


### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
